_posts/blog/2021-04-21-jshell-accumulo-feature.md

```diff
@@ -0,0 +1,141 @@
+---
+Title: JShell Accumulo Feature
+Author: R. Dane Magbuhos
+Reviewers: Christopher Tubbs, Keith Turner
+---
+
+## Overview
+First introduced in Java 9+, [JShell][JShell Doc] is an interactive Read-Evaluate-Print-Loop (REPL) 
+Java tool that assess user's inputed declarations, statements, and expressions and outputs 
+the results. This tool provides a convenient way to test out and execute quick tasks with Accumulo
+in the terminal.
+
+## Major Features
+* During Accumulo build, produces a default JShell script called `jshell-init.jsh` 
+containing up-to-date [Accumulo Java APIs][public APIs] and [AccumuloClient][client] 
+build implementation
+
+* Startup JShell with default or custom JShell script 
+
+* Both JShell start up options automatically import all relevant Java APIs 
+
+## Booting Up JShell Accumulo
+After installing and configuring the [latest Accumulo update][accumulo-repo]
+follow the steps below to startup JShell:
+
+1) Open up a terminal and navigate to Accumulo's home directory 
```

Comment:
Since "home directory" has a special meaning in Linux environments, it'd be better to refer to the Accumulo installation directory, if using the binary distribution tarball we provide. If they're using a distribution from a commercial vendor or other distributor, their experience may be different.